docs/UserGuide.adoc

```diff
@@ -347,29 +265,29 @@ Example:
 
 * `listIngredients` (returns chicken as 3rd index)
 +
-`restock i/3 q/10`
+`restockIngredient i/3 q/10`
 
 Adds 10 full chickens into inventory
 
 ==== Deplete Ingredient: `depleteIngredient`
 
 Deplete ingredient from inventory based on specified `INDEX` via `listIngredients`.
 
-Format: `depleteIngredient i/INGREDIENT_INDEX a/AMOUNT`
+Format: `depleteIngredient i/INGREDIENT_INDEX q/INGREDIENT_QUANTITY`
 
 Example:
 
 * `listIngredients` (returns chicken as 3rd index)
 +
-`depleteIngredient i/3 a/10`
+`depleteIngredient i/3 q/10`
 
 Depletes 10 full chickens from inventory
 
 ==== Delete Ingredient: `deleteIngredient`
 
 Delete ingredient from inventory based on specified `INDEX` via `listIngredients`. This command is irreversible.
```

Comment:
Why is the "this command is irreversible" here? 
Also, should probably make a reference that all corresponding ingredients will be deleted.
This is a non-obvious feature that might make users confused.